### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by JoramWells**

### CODE REVIEW

File: src/_actions/paymentActions.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const makePayment = (phone, amount) => async (dispatch) => {
       console.log(response)
     dispatch({ type: MAKE_PAYMENT_SUCCESS, payload: response });
 
-    })
+    }).catch(err=>console.log(err))
```
Comment (bramuel) [inline src/_actions/paymentActions.js:22]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch((err)=>console.log(err))
```

Comment (bramuel) [inline src/_actions/paymentActions.js:22]: Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch(err =>console.log(err))
```

File: src/_actions/paymentActions.js [hunk 2]
```diff
@@ -28,8 +33,17 @@ const makePayment = (phone, amount) => async (dispatch) => {
...
-    const { data } = await axios.post("/query", { requestID });
-    dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: data });
+    await axios.post("/query", { requestID })
+    .then(response=>{
+    // dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: response.data });
+    Cookie.set('confirmPaid', JSON.stringify(response.data),{
+      expires:1/28800
+
+    })
+    }).catch(err=>{
```
Comment (JoramWells) [inline src/_actions/paymentActions.js:43]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch((err)=>{
```

File: src/_actions/paymentActions.js [hunk 3]
```diff
@@ -28,8 +33,17 @@ const makePayment = (phone, amount) => async (dispatch) => {
 const confirmPayment = (requestID) => async (dispatch) => {
   dispatch({ type: CONFIRM_PAYMENT_REQUEST, payload: { requestID } });
   try {
-    const { data } = await axios.post("/query", { requestID });
-    dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: data });
+    await axios.post("/query", { requestID })
+    .then(response=>{
+    // dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: response.data });
+    Cookie.set('confirmPaid', JSON.stringify(response.data),{
```
Comment (JoramWells) [inline src/_actions/paymentActions.js:39]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    Cookie.set("confirmPaid", JSON.stringify(response.data),{
```

File: src/_actions/paymentActions.js [hunk 4]
```diff
@@ -10,16 +10,21 @@ import {
...
 } from "../_constants/paymentsConstant";
+const Cookie = require("js-cookie");
 
 const makePayment = (phone, amount) => async (dispatch) => {
   dispatch({ type: MAKE_PAYMENT_REQUEST, payload: { phone, amount } });
   try {
     await axios.post("/mpes/stk", { phone, amount })
     .then(response=>{
       console.log(response)
+      Cookie.set('paymentDetails', JSON.stringify(response.data),{
```
Comment (JoramWells) [inline src/_actions/paymentActions.js:21]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
      Cookie.set("paymentDetails", JSON.stringify(response.data),{
```

File: src/_actions/paymentActions.js [hunk 5]
```diff
@@ -10,16 +10,21 @@ import {
...
+        expires:1/28800
+
+      })
     dispatch({ type: MAKE_PAYMENT_SUCCESS, payload: response });
 
-    })
+    }).catch(err=>console.log(err))
```
Comment (JoramWells) [inline src/_actions/paymentActions.js:27]: Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch(err =>console.log(err))
```

File: src/_actions/paymentActions.js (hunk 2 above)
Comment (JoramWells) [inline src/_actions/paymentActions.js:43]: Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch(err =>{
```

File: src/_actions/paymentActions.js (hunk 5 above)
Comment (JoramWells) [inline src/_actions/paymentActions.js:27]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch((err)=>console.log(err))
```

**Pull request merged**